### PR TITLE
Adds the Presidential Alert command

### LINKED ISSUE #496: Presidential Alert image generator command
`s!presidentialalert <text>`

Example

Template: https://i.imgur.com/Zfnv5eg.jpg

Font is San Francisco, but can be easily replaced by Helvetica Neue.

### PR DESCRIPTION
**Pull request by pedrofracassi**
`s!presidentialalert <text>`

## Example
```
s!pa Mussum Ipsum, cacilds vidis litro abertis. Quem manda na minha terra sou euzis! Delegadis gente finis, bibendum egestas augue arcu ut est. Mais vale um bebadis conhecidiss, que um alcoolatra anonimis. Todo mundo vê os porris que eu tomo, mas ninguém vê os tombis que eu levo!
```
image

Closes #496

### CODE REVIEW

**Comment by Doges** [comment]
não tinha texto melhor não?

**Comment by justandras** [comment]
You should use the current date and time for the image.

**Comment by Doges** [comment]
@justandras but, which time we gonna use?

**Comment by justandras** [comment]
Server time?

**Comment by Doges** [comment]
yea but , if we are getting the server time, why we are putting a custom date, like, timezone sucks

File: src/commands/image-manipulation/presidentialalert.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    this.parameters = new CommandParameters(this,
+      new StringParameter({ full: true, required: true, missingError: 'commands:presidentialalert.missingText' })
+    )
+    this.requirements = new CommandRequirements(this, { canvasOnly: true })
+  }
+
+  async run ({ t, author, channel }, text) {
+    channel.startTyping()
+    const triggered = await CanvasTemplates.presidentialAlert(text)
```
Comment (Doges) [inline src/commands/image-manipulation/presidentialalert.js]:
```suggestion
    const presidential = await CanvasTemplates.presidentialAlert(text)
```

File: src/commands/image-manipulation/presidentialalert.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    channel.send(new Attachment(triggered, 'president.jpg'))
```
Comment (Doges) [inline src/commands/image-manipulation/presidentialalert.js]:
```suggestion
    channel.send(new Attachment(presidential, 'president.jpg'))
```

File: src/commands/image-manipulation/presidentialalert.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    this.parameters = new CommandParameters(this,
+      new StringParameter({ full: true, required: true, missingError: 'commands:presidentialalert.missingText' })
+    )
+    this.requirements = new CommandRequirements(this, { canvasOnly: true })
+  }
+
+  async run ({ t, author, channel }, text) {
+    channel.startTyping()
+    const presidential = await CanvasTemplates.presidentialAlert(text)
+    channel.send(new Attachment(presidential, 'president.jpg'))
```
Comment (dpaiv0) [inline src/commands/image-manipulation/presidentialalert.js]: ```channel.send(new Attachment(presidential, 'president.jpg')).then(() => channel.stopTyping())```

Comment (dpaiv0) [inline src/commands/image-manipulation/presidentialalert.js]:
```js
channel.send(new Attachment(presidential, 'president.jpg')).then(() => channel.stopTyping())
```

**Comment by pedrofracassi** [comment]
For some reason, the way you did it (using .then to .stopTyping) causes the bot to infinitely type sometimes. Putting stopTyping below the send method seems to have fixed that. Can we revert?

**Comment by dpaiv0** [comment]
oh, so that's what it did. ok, reverting

**Pull request merged**